samples/ocp/1.6.1/connectivity_client_v110.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: storage.dell.com/v1
+kind: ApexConnectivityClient
+metadata:
+  name: dell-connectivity-client
+  namespace: dell-connectivity-client
+spec:
+  client:
+    csmClientType: "apexConnectivityClient"
+    configVersion: v1.1.0
+    connectionTarget: connect-into.dell.com
+    forceRemoveClient: true
+    common:
+      name: connectivity-client-docker-k8s
+      image: registry.connect.redhat.com/dell-emc/connectivity-client-docker-k8s@sha256:204be30a60ee2864cf8e9a8a49f13efbee14222c78f27307288f59b5eb188158
+      imagePullPolicy: IfNotPresent
+    initContainers:
+      - name: connectivity-client-init
+        image: registry.connect.redhat.com/dell-emc/connectivity-client-docker-k8s@sha256:204be30a60ee2864cf8e9a8a49f13efbee14222c78f27307288f59b5eb188158
+        imagePullPolicy: IfNotPresent
+    sideCars:
+      - name: kubernetes-proxy
+        image: docker.io/bitnami/kubectl@sha256:e9d32369b107d0ceeee228c4dc19eff372c26009f29a2fc8f22327508f608542
+        imagePullPolicy: IfNotPresent
+      - name: cert-persister
+        image: docker.io/dellemc/connectivity-cert-persister-k8s@sha256:aaa935f3ba99a91bf5f00c53de5cd69e23716645ea9690f050c4ed80834661fc
```

Comment:
Are we not migrating to `quay.io`?

Reply:
Hi @harishp8889 this is for CSM 1.11.1. So that is not in scope.